xml/System/ReadOnlySpan`1.xml

```diff
@@ -688,9 +733,11 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Returns the string representation of this <see cref="T:System.ReadOnlySpan`1" /> object.</summary>
+        <returns>The string representation of this object.</returns>
```

Comment:
I don't think this is entirely accurate. The remark is more accurate.

The only time it returns the string representation is if T=char. For all other Ts, it returns the type and number of elements. We should say that instead since it is more generally true, and the remark should specify that for T=char, we return the string representation.

Thoughts?

Reply:
The same is true for Span.

Reply:
I think this is pretty standard wording for ToString. Remarks delves into more details about what that representation can be.